### PR TITLE
[#48289] improved controller for app password form

### PR DESCRIPTION
**Pull request by Kharonus**
- https://community.openproject.org/work_packages/48289
- simplified stimulus controller with adding action directly to switch component

### CODE REVIEW

**Review by mereghost — APPROVED** [review]
LGTM

**Comment by Kharonus** [comment]
I'll nevertheless wait for kabiru to accept it, as it alters his current work. ;)

File: modules/storages/app/views/storages/admin/storages/automatically_managed_project_folders/edit.html.erb [hunk 1]
```diff
@@ -51,12 +51,15 @@ See COPYRIGHT and LICENSE files for more details.
...
-      <label class="form--label -flex" data-storages--automatically-managed-project-folders-form-target="automaticallyManagedSwitch">
+      <label class="form--label -flex">
         <%= angular_component_tag 'spot-switch',
                                   inputs: {
                                     id: "storages_#{@storage.short_provider_type}_storage_automatically_managed",
                                     name: "storages_#{@storage.short_provider_type}_storage[automatically_managed]",
                                     checked: @storage.automatically_managed?,
+                                  },
+                                  data: {
+                                    action: 'change->storages--automatically-managed-project-folders-form#updateDisplay',
```
Comment (akabiru) [inline modules/storages/app/views/storages/admin/storages/automatically_managed_project_folders/edit.html.erb:62]: Really nice one here 🧑‍🎨

**Review by akabiru — APPROVED** [review]
Thank you @Kharonus 🤩  this is such a refined improvement! 🎨

**Pull request merged**